airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -100,8 +100,8 @@ def close(self) -> None:
             with open(local_loc) as logfile:
                 log = logfile.read()
             self.wasb_write(log, remote_loc, append=True)
-
-            if self.delete_local_copy:
+            keep_local = conf.getboolean('logging', 'KEEP_LOCAL_LOGS')
+            if self.delete_local_copy or not keep_local:
```

Comment:
I wonder if `delete_local_copy` is still needed now that you have introduced this global behaviour?

Reply:
My only issue was keeping compatibility for those who right now rely on that flag, not sure what's the policy regarding that.